lib/landrush/cap/all/read_host_visible_ip_address.rb

```diff
@@ -0,0 +1,49 @@
+module Landrush
+  module Cap
+    module All
+      module ReadHostVisibleIpAddress
+        def self.filter_addresses(addresses)
+          unless @machine.config.landrush.exclude.nil?
+            re = Regexp.union(@machine.config.landrush.exclude)
+
+            addresses = addresses.select do |addr|
+              !addr['name'].match(re)
+            end
+          end
+
+          addresses
+        end
+
+        def self.read_host_visible_ip_address(machine)
+          @machine = machine
+
+          addr      = nil
+          addresses = machine.guest.capability(:landrush_ip_get)
```

Comment:
just so that I understand correctly, this is the part which "requests" the landrush-ip capability to which we have now a dependency in the Gemfile and which you release from https://github.com/Werelds/landrush-ip, right?